### ISSUE TITLE
ESP32-S3-Korvo-2: Generic LCD initialization function is not available (BSP-269)

### ISSUE THREAD

**Issue by TTauriainenQt** [espressif/esp-bsp#121]
Public function for generic LCD initialization is not available.

Proposed API-function prototype:
`esp_lcd_panel_handle_t bsp_lcd_init(esp_lcd_panel_io_color_trans_done_cb_t on_color_trans_done);`

Functionality would be same as in `static lv_disp_t *bsp_display_lcd_init(void)`
lines 206 - 258 + setting callback function to `esp_lcd_panel_io_spi_config_t` + returning pointer to the lcd panel handle.

https://github.com/espressif/esp-bsp/blob/ee79fd887a1a331bf9eb3935aa0e602780aa80fc/esp32_s3_korvo_2/esp32_s3_korvo_2.c#L206-L258

Alternative proposal:

Add two public API-function similar to ESP-BOX LCD BSP.

```
esp_lcd_panel_handle_t bsp_lcd_init(void);
void bsp_lcd_set_cb(esp_lcd_panel_io_color_trans_done_cb_t on_color_trans_done);
```

For reference
https://github.com/espressif/esp-box/blob/master/components/bsp/src/peripherals/bsp_lcd.c#L115
https://github.com/espressif/esp-box/blob/master/components/bsp/src/peripherals/bsp_lcd.c#L227

**Comment by tore-espressif** [comment]
@TTauriainenQt do you need deinitialization function too?
That would free all the memory and HW resources?

**Comment by tore-espressif** [comment]
Closed by #136
